akka-persistence-typed/src/main/scala/akka/persistence/typed/crdt/OpCrdt.scala

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.persistence.typed.crdt
+import akka.annotation.ApiMayChange
+
+@ApiMayChange
+trait OpCrdt[Operation] { self =>
```

Comment:
I'm guessing this isn't really usable from Java because of the type bound trixery but maybe that does not matter because user is not supposed to extend?

Reply:
added do not inherit